data/migrations/20190501115744_invites.js

```diff
@@ -0,0 +1,29 @@
+exports.up = function(knex, Promise) {
+	return knex.schema.createTable('invites', table => {
+		table
+			.integer('manager_id')
+			.unsigned()
+			.notNullable()
+			.references('users.user_id');
+
+		table
+			.integer('cleaner_id')
+			.unsigned()
+			.references('users.user_id');
+
+		table.string('email', 128).notNullable();
+
+		table
+			.string('inviteCode', 16)
+			.notNullable()
+			.unique();
+
+		table.boolean('rejected');
```

Comment:
We talked about this but a "status" integer could be better.
0 = pending
1 = accepted
2 = rejected